modules/nf-core/isoseq/cluster/tests/main.nf.test

```diff
@@ -0,0 +1,46 @@
+nextflow_process {
+
+    name "Test Process ISOSEQ_CLUSTER"
+    script "../main.nf"
+    config "./nextflow.config"
+    process "ISOSEQ_CLUSTER"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "isoseq"
+    tag "isoseq/cluster"
+
+    test("Main test") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test' ], // meta map
+                    file(params.test_data['homo_sapiens']['pacbio']['refine'], checkIfExists: true),
```

Comment:
Can you update the file paths according to #5281

Reply:
Done